crates/runtime/src/libcalls.rs

```diff
@@ -185,14 +185,14 @@ pub extern "C" fn wasmtime_f64_nearest(x: f64) -> f64 {
 }
 
 /// Implementation of memory.grow for locally-defined 32-bit memories.
-pub unsafe extern "C" fn wasmtime_memory32_grow(
+pub unsafe extern "C" fn memory32_grow(
     vmctx: *mut VMContext,
     delta: u64,
     memory_index: u32,
-) -> usize {
+) -> *mut u8 {
```

Comment:
This function doesn't return an actual pointer, right?

Reply:
Correct, this is just how the type-checking with the macro worked out.